src/defaults.js

```diff
@@ -0,0 +1,31 @@
+let logsFolder = (() => {
+  let folder = './logs/';
+  if (typeof process !== 'undefined' && process.env.NODE_ENV === 'test') {
+    folder = './test_logs/'
+  }
+  folder = process.env.PARSE_SERVER_LOGS_FOLDER || folder;
+  return folder;
+})();
+
+let { verbose, level } = (() => {
+  let verbose = process.env.VERBOSE ? true : false;
+  return { verbose, level: verbose ? 'verbose' : undefined }
+})();
+
+export default {
+  DefaultMongoURI: 'mongodb://localhost:27017/parse',
+  jsonLogs: process.env.JSON_LOGS || false,
+  logsFolder,
+  verbose,
+  level,
+  silent: false,
+  enableAnonymousUsers: true,
+  allowClientClassCreation: true,
+  maxUploadSize: '20mb',
+  verifyUserEmails: false,
+  preventLoginWithUnverifiedEmail: false,
+  sessionLength: 31536000,
+  expireInactiveSessions: true,
+  revokeSessionOnPasswordReset: true,
+  schemaCacheTTL: 5
```

Comment:
I think this value is incorrect (carried over from previous code), cache times are in milliseconds in InMemoryCache so should be 5000 if 5 seconds is the intention.
SchemaCache has a similar bug in the constructor but could be a separate PR